core/src/main/scala/org/dbpedia/extraction/util/WikiApi.scala

```diff
@@ -218,7 +218,12 @@ class WikiApi(url: URL, language: Language)
         {
             try
             {
-                val reader = new URL(url + params).openStream()
+                val connection = new URL(url + params).openConnection()
+                connection.setRequestProperty(
+                    "User-Agent",
```

Comment:
now that we moved the wikipedia abstract extractor in Core i think we should move this to configuration as well.
since people might use that extractor and the live issue shows up on the Wikipedia's logs

see
1: https://github.com/dbpedia/extraction-framework/blob/6f4a4acc7719b3cd69acdd503bf40bcbf92db862/dump/pom.xml#L81-L81

and 2: https://github.com/dbpedia/extraction-framework/blob/bb7ec4d4910127f0a576f68270ff4e32707146d1/core/src/main/scala/org/dbpedia/extraction/config/mappings/InfoboxExtractorConfig.scala#L30-L34